src/manager.ts

```diff
@@ -127,31 +148,34 @@ export class SecretsManager implements ISecretsManager {
    * Actually fetch the secret from the connector.
    */
   private async _get(id: string): Promise<ISecret | undefined> {
```

Comment:
I'm also worried about this kind of method, that doesn't check the token.
This method is still available in javascript, even if private in typescript context.

Reply:
you can move it into the Private namespace?

Reply:
:+1: I can make them static and move them to the Private namespace.